native/SampleApps/SmartSyncExplorer/SmartSyncExplorer.xcodeproj/project.pbxproj

```diff
@@ -76,7 +77,7 @@
 		BCC8F2271D46C01900171499 /* HostPage.swift in Sources */ = {isa = PBXBuildFile; fileRef = BCC8F2261D46C01900171499 /* HostPage.swift */; };
 		CE2B8D671CE932D600C6FC6A /* SalesforceAnalytics.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = CE2B8D4C1CE932BD00C6FC6A /* SalesforceAnalytics.framework */; };
 		CE6AB39B1C10C4F50012125E /* libz.tbd in Frameworks */ = {isa = PBXBuildFile; fileRef = CE6AB39A1C10C4F50012125E /* libz.tbd */; };
-		CEF4A2EE1D9491010025EDF0 /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = CEF4A2EC1D9491010025EDF0 /* LaunchScreen.storyboard */; };
+		CEF4A2EE1D9491010025EDF0 /* (null) in Resources */ = {isa = PBXBuildFile; };
```

Comment:
1. Go to SmartSyncExplorer project settings.
2. Select the SmartSyncExplorer target.
3. Select Build Phases.
4. Expand Copy Bundle Resources.
5. Remove the empty entry in that list.

Voila.  Null project item goes away. 🙂